bloom-instance/services/partner-site/service.tf

```diff
@@ -0,0 +1,15 @@
+
+module "service" {
+  source = "../base-service"
+
+  name_prefix = var.name_prefix
+  #service_name       = var.service_definition.name
```

Comment:
Is there a reason why this is still commented out?

Reply:
I'm considering having a map of values rather than a list, in which case I would need to pass it in as a separate arg rather than as part of the value object.  The main benefit of doing the map approach is that it forces unique values for names, which would otherwise be a potential source of human error.  Leaving this commented makes the switch easier later on.  If it turns out I don't need it, I can just clean it out later.